cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -249,12 +248,12 @@ static void usage(boolean error) {
     if (OSDetector.isWindows()) {
       out.println("  cdap.bat [options]");
     } else {
-      out.println("  ./cdap.sh [options]");
+      out.println("  cdap.sh [options]");
```

Comment:
it is kind of weird that the Java code prints the options for the shell/bat script. But since it does, should it mention --enable-debug?